snooty/test_parser.py

```diff
@@ -141,6 +141,7 @@ def test_card() -> None:
 .. card-group::
    :columns: 3
    :layout: carousel
+   :type: drivers
```

Comment:
when i remove this option, i'm not seeing the `card_group_type` to be set to `default` here. is this intended? otherwise LGTM!

Reply:
@seungpark that was the intent, if the options aren't set, there it should be `undefined`, but the optional option could either be `default` or `drivers`